### PR TITLE
fix: [PL-43404]: Add path urls and operation ids for org and account scope

### PR DESCRIPTION
**Pull request by jennyjames21**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/56974)

### CODE REVIEW

**Comment by jennyjames21** [comment]
trigger uts

**Comment by jennyjames21** [comment]
trigger ut1

**Comment by jennyjames21** [comment]
trigger ut2

**Comment by jennyjames21** [comment]
trigger ut3

File: platform-service/modules/notification-service/contracts/openapi/v1/openapi.yaml [hunk 1]
```diff
@@ -205,6 +205,402 @@ paths:
...
+      security:
+        - x-api-key: []
+      tags:
+        - Notification Rules
+      x-internal: true
+    parameters:
+      - $ref: '#/components/parameters/OrgPathParam'
+  /v1/notification-rules:
+    get:
+      summary: List Notification rules at org level
```
Comment (mankrit-singh) [inline platform-service/modules/notification-service/contracts/openapi/v1/openapi.yaml:332]: *account

**Review by mankrit-singh — COMMENTED** [review]
important- there should be a different tag for each scope eg. Account Secrets, Org Secrets, Project Secrets

Otherwise it could lead to jax-rs routing issues in the future

File: platform-service/modules/notification-service/contracts/openapi/v1/openapi.yaml (hunk 1 above)
Comment (jennyjames21) [reply platform-service/modules/notification-service/contracts/openapi/v1/openapi.yaml]: updated

**Comment by jennyjames21** [comment]
trigger uts

**Comment by jennyjames21** [comment]
> trigger ut1

yes,

**Comment by jennyjames21** [comment]
trigger ut1

**Comment by jennyjames21** [comment]
trigger ut2

**Comment by jennyjames21** [comment]
trigger ut1

**Comment by jennyjames21** [comment]
trigger ut2

**Comment by jennyjames21** [comment]
trigger ut3

**Comment by jennyjames21** [comment]
trigger ut4

**Comment by jennyjames21** [comment]
trigger uts

**Comment by jennyjames21** [comment]
trigger uts

**Review by mankrit-singh — APPROVED** [review]

**Comment by jennyjames21** [comment]
trigger ut6

**Comment by jennyjames21** [comment]
trigger ut7

**Comment by jennyjames21** [comment]
trigger ut9

**Pull request merged**